src/methods.jl

```diff
@@ -92,9 +103,12 @@ julia> subtract_overscan(frame, "[4:5, 1:1]", dims = 2)
 ```
 
 # See Also
-* [`subtract_overscan!`](@ref)
+[`subtract_overscan!`](@ref)
 """
-subtract_overscan(frame::AbstractArray, idxs; dims = axes_min_length(idxs)) = subtract_overscan!(deepcopy(frame), idxs, dims = dims)
+subtract_overscan(frame, idxs; kwargs...) = subtract_overscan!(deepcopy(frame), idxs; kwargs...)
+subtract_overscan(frame::ImageHDU, idxs; kwargs...) = subtract_overscan!(getdata(frame), idxs; kwargs...)
```

Comment:
Please add a docstring here
```julia
"""
	subtract_overscan(::FITSIO.ImageHDU, idxs; [dims])
	subtract_overscan(filename, idxs; hdu=1, [dims])

Load a FITS file or HDU before subtracting the overscan region. If `idxs` is a symbol it will be read from the FITS header with that key (case sensitive).
"""
```